src/chapter3/02-writing-the-api.md

```diff
@@ -0,0 +1,320 @@
+# Writing the API
+
+## Our tasks
+
+For now we will store our tasks in a global variables named `tasks`.
```

Comment:
it's not a global variable; it would be like this `global.tasks = ...`

Reply:
fixed